tools/release-posts/templates/rc-release.ejs

```diff
@@ -0,0 +1,121 @@
+<!-- wp:paragraph -->
+<p>
+	Release Candidate <%= rcNumber %> for the <%=
+	finalReleaseDate.toLocaleDateString('en-US', {month: 'long', day:
+	'numeric'}) %> release of WooCommerce is now available for testing! You can
+	either
+	<a
+		href="https://downloads.wordpress.org/plugin/woocommerce.<%= version.version %>.zip"
+		target="_blank"
+		rel="noreferrer noopener"
+		>download it directly from WordPress.org</a
+	>
+	or install our
+	<a
+		rel="noreferrer noopener"
+		href="https://woocommerce.wordpress.com/2018/07/30/woocommerce-beta-tester-2-0-0/"
+		target="_blank"
+		>WooCommerce Beta Tester Plugin</a
+	>.
+</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:heading -->
+<h2 class="wp-block-heading">What's New?</h2>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph -->
+<p>
+	Since the release of
+	<a href="<%= lastReleasePostUrl %>"><%= prettyPreviousVersion %></a>, the
+	following changes have been made:
+</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:list -->
+<ul>
+	<!-- wp:list-item -->
+	<li>List the changes to the release here.</li>
+	<!-- /wp:list-item -->
+</ul>
+<!-- /wp:list -->
+
+<!-- wp:heading -->
+<h2 class="wp-block-heading">Actions and Filters</h2>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph -->
+<p>Insert action and filter changes here.</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:heading -->
+<h2 class="wp-block-heading">Template Changes</h2>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph -->
+<p>Insert template changes here.</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:heading {"anchor":"release-schedule"} -->
+<h2 class="wp-block-heading" id="release-schedule">Release Schedule</h2>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph -->
+<p>
+	We're still on track for our planned <%=
+	finalReleaseDate.toLocaleDateString('en-US', {month: 'long', day:
+	'numeric'}) %> release of WooCommerce <%= version.major %>.<%= version.minor
+	%>.
+</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:heading {"anchor":"testing"} -->
+<h2 class="wp-block-heading" id="testing">Testing</h2>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph -->
+<p>
+	If you'd like to dive in and help test this new release, our handy&nbsp;<a
```

Comment:
There's a space before the "If"

<img width="205" alt="image" src="https://user-images.githubusercontent.com/1922453/226230264-e245f2ee-4fce-4e58-85e6-9710e6c84f06.png">

Reply:
Looking further, every paragraph beginning in the "Testing", "What's New?" and "Release Schedule" has an extra space too.